crates/acap-ssh-utils/src/lib.rs

```diff
@@ -92,16 +117,32 @@ pub fn run_other<S: AsRef<str>>(
     env: &[(S, S)],
     args: &[&str],
 ) -> anyhow::Result<()> {
-    let sftp = session.sftp()?;
-    let mut channel = session.channel_session()?;
+    let tmp = RemoteCommand::new(None::<&str>, None::<&[(&str, &str)]>, "mktemp -u", None)
+        .exec_capture_stdout(session)?;
 
-    let tmp = RemoteCommand::new(None, None::<&[(&str, &str)]>, "mktemp", None)
-        .exec_capture_stdout(&mut channel)?;
+    // The output from `mktemp -u` contains a trailing '\n'
+    let path = std::str::from_utf8(&tmp)?.strip_suffix('\n').unwrap();
```

Comment:
This is essentially parsing input from a source that we don't control, so I don't think we should panic if it doesn't conform to our expectations.